playbooks/group_vars/webportals.yml

```diff
@@ -49,6 +49,23 @@ local_networks_denied:
   - "198.18.0.0/15"
   - "169.254.0.0/16"
 
+# Iptables: Tor exit nodes blocking
+block_tor_exit_nodes: True
+block_tor_exit_nodes_lists:
+  torproject:
+    url: "https://check.torproject.org/torbulkexitlist?ip="
+    filename: "torproject-exitnodelist"
+    ipset_ipv4: "torproject-exitnodelist-ipv4"
+  dan:
+    url: "https://www.dan.me.uk/torlist/?exit"
+    filename: "dan-exitnodelist"
+    ipset_ipv4: "dan-exitnodelist-ipv4"
+    ipset_ipv6: "dan-exitnodelist-ipv6"
+  # Downloads from dan.me.uk are limited to once per 30 mins (= 1800 secs), we
+  # do not redownload the given number of seconds to prevent errors loading
+  # error message instead of IP list to ipset
+  do_not_redownload_within_secs: 2000
```

Comment:
nit: It would be nice to add a comment here explaining that we're caching the result in `/tmp` in order to avoid redownloading it.

Reply:
The redownloads are handled by the shell script,
so user or cronjob can run the script without any restrictions (like 3x in 1 minute or every 5 mins)

During download a timestamp file is saved, if we download every 5 mins, the script will skip downloading until (timestamp in a file + `do_not_redownload_within_secs`) < now time.

Reply:
Yes, I noticed that. What I mean by this comment is that it would be nice to explain that here with a single sentence, so people won't wonder what happens and go to read the shell script in order to find out.